### PR TITLE
VR-3616 test cases

### PR DESCRIPTION
**Pull request by evgeniilezhnin**

### CODE REVIEW

File: backend/src/test/java/ai/verta/modeldb/CommitTest.java [hunk 1]
```diff
@@ -714,6 +714,8 @@ public void getCommitComponentTest() {
     LOGGER.info("Get commit blob test end................................");
   }
 
+  private static long time = Calendar.getInstance().getTimeInMillis();
```
Comment (anandjakhaniya) [inline backend/src/test/java/ai/verta/modeldb/CommitTest.java]: please add this variable on top of the class

Comment (evgeniilezhnin) [reply backend/src/test/java/ai/verta/modeldb/CommitTest.java]: fixed

File: backend/src/test/java/ai/verta/modeldb/DiffTest.java [hunk 2]
```diff
@@ -107,6 +136,19 @@ public static void setServerAndService() throws Exception {
     }
   }
 
+  private final int blobType;
+  private final int commitType;
+
+  @Parameters
+  public static Collection<Object[]> data() {
```
Comment (ravishetye) [inline backend/src/test/java/ai/verta/modeldb/DiffTest.java:149]: add a comment what these numbers indicate are for quick reference. Also if you can summarize what combinations are being tested either at the top of the file or at individual test , that will be super helpful

**Review by ravishetye — APPROVED** [review]
Minor comment added for documentation, but approved otherwise

File: backend/src/test/java/ai/verta/modeldb/DiffTest.java (hunk 2 above)
Comment (evgeniilezhnin) [reply backend/src/test/java/ai/verta/modeldb/DiffTest.java:149]: fixed

**Pull request merged**